README.md

```diff
@@ -12,32 +12,43 @@ If possible, it is preferable to store image content within this repository at `
 
 ### How to add a new page
 
-Let’s consider an example. Let’s imagine we want to add a page called `Top 5 mistakes` in section `Basics` after the `Getting Started` page. This page already exists, so you can already see the results in WLED Docs, but let’s imagine the page is not there and we will go steps needed to add it:
+Let’s consider an example. Let’s imagine we want to add a page called `Top 5 mistakes` in section `Basics` after the `Getting Started` page. This page already exists, so you can already see the results in WLED Docs, but let’s imagine the page is not there and we will go over the steps needed to add it:
 
-1)	Create a file that should contain the documentation. As we want to place the new page in section `Basics` it makes sense to put this file in subfolder `docs/basics/`. In our example we call the file `top5_mistakes.md`. You can use another .md file as an example. Important is to have the following few lines at the beginning of the file:
-```
----
-title: TOP 5 mistakes
-hide:
-  # - navigation
-  # - toc
----
-```
-Don’t forget to adapt the title.
```

Comment:
Fixing the indentation here makes this look nicer when rendered:

**Before:**
![image](https://github.com/Aircoookie/WLED-Docs/assets/12214617/168a5f32-08d1-4a69-a995-3388115caa60)

**After:**
![image](https://github.com/Aircoookie/WLED-Docs/assets/12214617/8c24057a-fc80-48a0-92e8-0d77a30fc5a0)